src/Engine/ProtoCore/DSASM/Executive.cs

```diff
@@ -1422,7 +1428,6 @@ private int UpdateGraph(int exprUID, bool isSSAAssign)
                 }
                 gnode.isActive = false;
             }
-            return reachableGraphNodes.Count;
```

Comment:
Why don't we keep this `return `statement, that way we can just say `return GraphUpdateImpl(...);` above.

Reply:
Yes that makes sence.  I will update that.